app/models/user.js

```diff
@@ -0,0 +1,10 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  name: DS.attr(),
+  email: DS.attr(),
+  password: DS.attr(),
+  city: DS.attr(),
+  state: DS.attr(),
+  radius: DS.attr()
```

Comment:
We can talk about it, but it feels like radius should go in their profile... Not necessarily something we need to address now...

Reply:
Sounds good, maybe we can chat with the backend team about it too. They may have some fresh insight with their perspective.